src/body_tracking_tf_broadcaster.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "rclcpp/rclcpp.hpp"
+#include <visualization_msgs/msg/marker_array.hpp>
+#include <geometry_msgs/msg/transform_stamped.hpp>
+#include <geometry_msgs/msg/pose.hpp>
+#include <tf2_ros/transform_broadcaster.h>
+
+
+using std::placeholders::_1;
+
+
+class BodyTrackingTFBroadcaster : public rclcpp::Node
+{
+  public:
+    BodyTrackingTFBroadcaster()
+    : Node("tf_body_tracking_broadcaster")
+    {
+        subscription_ = this->create_subscription<visualization_msgs::msg::MarkerArray>(
+        "/body_tracking_data", 10, std::bind(&BodyTrackingTFBroadcaster::callback, this, _1));
+
+
+        // Initialize map with id and corresponding body part
+        int index_start = 100;
```

Comment:
How would this handle multiple bodies being tracked by the SDK?

Reply:
Good point, we'll have to go through all the markers.